### DISCUSSION TITLE
Subdomain Not Working

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3460] [Help / Questions]
**Question**
I set up my Wiki.js site with a Marketplace App through Digital Ocean. My issue is my domain will not work with the subdomain, so for example instead of `wiki.example.com`, it only works with `example.com`. 

I have my domain, `example.com`, being forwarded through DO, and `wiki.example.com` on Wiki.js, however it's only working with the original domain, not the subdomain.

**Host Info (please complete the following information):**
OS: Ubuntu 18.04, Docker
Wiki.js version: 2.5.159
Database engine: PostgreSQL 11.8

**Comment by Smankusors** [comment]
What do you mean by "not working"? Please be more specific.

**Comment by ghost** [comment]
> What do you mean by "not working"? Please be more specific.

I did: I specified `wiki.example.com` as the site URL, however I can only get to my site through `example.com`, not through the subdomain.

**Comment by Smankusors** [comment]
Unfortunately it's still vague, it's not specific enough. When you visit wiki.example.com, did you get, connection timed out? DNS not resolved? Connection reset? Not found? Or something else?

**Comment by Frankstar** [comment]
if its working with example.com and not with wiki.example.com the problem probably is an DNS / redirect issue.
i have no clue about Digital Ocean tho ... 
can you show us the configuration?

do both URLs resolve to the same IP? (i guess so)
There shouldn't be an difference server-side, if you dont change Ports and if there isn't a  Proxy or another webserver between.
(infront of the node server)
